spec/unit/util/dsc/local_configuration_manager_spec.rb

```diff
@@ -114,23 +142,23 @@
         let(:lcm_standard_error) { "Abort, Retry, Fail?" }
         let(:lcm_cmdlet_success) { false }
 
-        it "should log a warning" do
+        it "log a warning" do
           expect(Chef::Log).to receive(:warn).at_least(:once)
           expect(lcm).to receive(:dsc_module_import_failure?).and_call_original
           expect { lcm.test_configuration("config", {}) }.not_to raise_error
         end
       end
     end
 
-    it "should identify a correctly formatted error message as a resource import failure" do
+    it "identify a correctly formatted error message as a resource import failure" do
       expect(lcm.send(:dsc_module_import_failure?, dsc_resource_import_failure_output)).to be(true)
     end
 
-    it "should not identify an incorrectly formatted error message as a resource import failure" do
```

Comment:
`does` not identify an incorrectly formatted error message as a resource import failure.

Applicable to the specs below as well.